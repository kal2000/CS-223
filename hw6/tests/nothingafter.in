Aarhus,
Belarus
Aarhus
Aarhus,
Aarhus,
Belarus
Aarhus,
Aarhus, again
Aarhus
