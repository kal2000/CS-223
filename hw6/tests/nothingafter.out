Aarhus,
Belarus
Aarhus
Aarhus,
ibid.
op. cit. Belarus
op. cit. Aarhus
Aarhus, again
Aarhus
